scripts/exgfs_wave_prdgen_bulls.sh

```diff
@@ -115,8 +115,8 @@
    echo "${msg}"
    set_trace
    echo "${RUN} wave prdgen ${date} ${cycle} : Bulletin header data file missing."
-   err=3;export err;err_chk
-   exit ${err}
+   export err=3
+   err_chk
```

Comment:
Should the `msg` on line 108 and/or `echo` on line 117 be added to the `err_chk`?